<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/app.png</normaloff>icons/app.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="12">
    <widget class="QTreeWidget" name="baseTree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Names</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Access</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Modification</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="undoButton">
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/left.png</normaloff>icons/operations/left.png</iconset>
     </property>
     <property name="shortcut">
      <string>Backspace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QToolButton" name="goButton">
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/go.png</normaloff>icons/operations/go.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="7">
    <widget class="QLineEdit" name="commandArea"/>
   </item>
   <item row="0" column="4" colspan="7">
    <widget class="QLineEdit" name="directoryLabel"/>
   </item>
   <item row="7" column="2">
    <widget class="QToolButton" name="newFolderButton">
     <property name="text">
      <string>N</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/new folder.png</normaloff>icons/operations/new folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="removeButton">
     <property name="text">
      <string>R</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/remove.png</normaloff>icons/operations/remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QToolButton" name="renameButton">
     <property name="text">
      <string>R</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/rename.png</normaloff>icons/operations/rename.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="homeButton">
     <property name="text">
      <string>H</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/home.png</normaloff>icons/operations/home.png</iconset>
     </property>
     <property name="shortcut">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="7" column="11">
    <widget class="QToolButton" name="execButton">
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/execute.png</normaloff>icons/operations/execute.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="redoButton">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/operations/right.png</normaloff>icons/operations/right.png</iconset>
     </property>
     <property name="shortcut">
      <string>Shift+Backspace</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/open folder.png</normaloff>icons/open folder.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QToolButton" name="toolButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/console.png</normaloff>icons/console.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
